Win32/Prj/libpcap.vcxproj.filters

```diff
@@ -43,7 +40,22 @@
     <ClCompile Include="..\..\savefile.c">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="..\..\scanner.c">
+    <ClCompile Include="..\..\..\Win32-Extensions\grammar.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="..\..\..\Win32-Extensions\scanner.c">
```

Comment:
Again, the build process should run Flex and Bison, which should put those source files in the main source directory, so those two changes shouldn't be necessary.